### PR TITLE
Indicate that Grid Layout is obsolete

### PR DESCRIPTION
**Pull request by Wiggee11**
Suggest marking Grid Layout and Nested Content as obsolete and directing users to replacement Block Grid and Block List respectively.

### CODE REVIEW

File: umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/grid-layout/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Grid Layout
+# Grid Layout (Legacy)
+
+{% hint style="info" %} We highly recommend that you use the [Block Grid](../../block-editor/block-grid-editor.md) instead.
+
+Grid Layout has been marked as obsolete and will be removed from future versions of Umbraco. {% endhint %}
```
Comment (sofietoft) [inline umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/grid-layout/README.md:5]: As we can't give a specific version number as to when we will remove the property, how about we write something like this instead:

```suggestion
Grid Layout has been marked as obsolete and development on the property editor has been discontinued. {% endhint %}
```

**Review by sofietoft — COMMENTED** [review]
This is a great addition @Wiggee11 !
Makes sense to direct people to the new editors instead 💪 

I've added a single suggestion - let me know what you think!

**Comment by Wiggee11** [comment]
Sensbile suggestion. I didn't really know how to phrase it!

**Comment by sofietoft** [comment]
Yeah, it's definitely a tricky one to get "just right" 😅 
Glad you agreed with my suggestions.

Let's get this one merged in 💪

**Pull request merged**